src/controllers/list.js

```diff
@@ -791,21 +791,37 @@ class ItemsView {
 
                 const itemType = item ? item.Type : null;
 
-                if (itemType === 'MusicGenre' || params.type !== 'Programs' && itemType !== 'Channel') {
+                if ((itemType === 'MusicGenre' || params.type !== 'Programs' && itemType !== 'Channel')
```

Comment:
The list view does not seem to be used for these, but I was afraid to remove this condition. :sweat_smile: